pkg/nix/nix.go

```
@@ -31,22 +32,40 @@ func IsInstalled() bool {
 // dependencies can be either a package name ex. php or a path to .nix file
 // nixpkgs can be empty which means it will use local nixpkgs channel
 // or a link to desired revision ex. https://github.com/NixOS/nixpkgs/archive/eeefd01d4f630fcbab6588fe3e7fffe0690fbb20.tar.gz
-func BuildDependencies(deps []Dependency) (DependenciesToStorePathMap, error) {
+func BuildDependencies(deps []Dependency) (_ DependenciesToStorePathMap, err error) {
+	defer errz.Recover(&err)
+
+	c, err := NewCacheStore()
+	errz.Fatal(err)
+	defer func() {
+		err = c.Close()
```

Would be ok for me to ignore the error.